<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editSearch">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFind">
        <property name="text">
         <string>Искать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeSelect">
        <item>
         <property name="text">
          <string>CSV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>JSON</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MyBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="ID">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editID"/>
      </item>
      <item>
       <widget class="QLabel" name="Model">
        <property name="text">
         <string>Модель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editModel"/>
      </item>
      <item>
       <widget class="QLabel" name="Year">
        <property name="text">
         <string>Год:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editYear"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Цвет:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboColor">
        <item>
         <property name="text">
          <string>Красный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Зеленый</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Синий</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Черный</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyBrowser</class>
   <extends>QTextBrowser</extends>
   <header>mybrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>findCars()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addCars()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>737</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openCars()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findCars()</slot>
  <slot>addCars()</slot>
  <slot>openCars()</slot>
 </slots>
</ui>
